### PR TITLE
Cleanup, part 3

### PR DESCRIPTION
**Pull request by mimizh2418**
Clean up some variable names and kills fake subsystems and other misc cleanup things

### CODE REVIEW

File: src/main/java/org/team1540/robot2024/subsystems/indexer/Indexer.java [hunk 1]
```diff
@@ -21,6 +22,8 @@ public class Indexer extends SubsystemBase {
     private final LoggedTunableNumber kI = new LoggedTunableNumber("Indexer/kI", FEEDER_KI);
     private final LoggedTunableNumber kD = new LoggedTunableNumber("Indexer/kD", FEEDER_KD);
 
+    private final double feederSetpointRPS = 0.0;
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2024/subsystems/indexer/Indexer.java:25]: Why not rpm

File: src/main/java/org/team1540/robot2024/subsystems/indexer/Indexer.java [hunk 2]
```diff
@@ -68,14 +71,41 @@ public boolean isNoteStaged() {
         return inputs.noteInIntake;
     }
 
-    public void setFeederVelocity(double setpointRPM) {
-        io.setFeederVelocity(setpointRPM);
+    public void setFeederVelocity(double setpointRPS) {
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2024/subsystems/indexer/Indexer.java:74]: Again, why not rpm

File: src/main/java/org/team1540/robot2024/subsystems/indexer/Indexer.java [hunk 3]
```diff
@@ -68,14 +71,41 @@ public boolean isNoteStaged() {
...
+        io.setIntakeVoltage(0);
+        io.setFeederVoltage(0);
+    }
+
+    @AutoLogOutput(key = "Intake/Feeder/setpointRPS")
+    public double getFeederVelocitySetpoint() {
+        return feederSetpointRPS;
+    }
+
+    @AutoLogOutput(key = "Intake/Feeder/setpointRPS")
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2024/subsystems/indexer/Indexer.java:104]: 🙅

**Review by rutmanz — CHANGES_REQUESTED** [review]
Why is everything RPS and not RPM?

File: src/main/java/org/team1540/robot2024/subsystems/indexer/IndexerIO.java [hunk 4]
```diff
@@ -6,17 +6,15 @@ public interface IndexerIO {
...
     class IndexerIOInputs {
-        public double intakeVoltage;
-        public double intakeCurrentAmps;
-        public double intakeVelocityRPM;
-        public double intakeTempCelsius;
-        public double feederVoltage;
-        public double feederCurrentAmps;
-        public double feederVelocityRPM;
-        public double feederTempCelsius;
+        public double intakeVoltage = 0.0;
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2024/subsystems/indexer/IndexerIO.java:9]: don't doubles get initialized to 0 by default?

File: src/main/java/org/team1540/robot2024/subsystems/shooter/Shooter.java [hunk 5]
```diff
@@ -261,12 +260,12 @@ public Command spinUpAndSetPivotPosition(Supplier<Double> leftSetpoint, Supplier
         );
     }
 
-    @AutoLogOutput
+    @AutoLogOutput(key = "Shooter/Flywheels/leftSetpointRPM")
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2024/subsystems/shooter/Shooter.java:263]: ok to be clear, we now have AdvantageScope/RealOutputs/Shooter/Flywheels, AdvantageScope/Shooter/Flywheels, SmartDashboard/Shooter/Flywheels, and SmartDashboard/TunableNumbers/Shooter/Flywheels. May be useful to think about consolidating some of those

**Review by rutmanz — APPROVED** [review]

File: src/main/java/org/team1540/robot2024/subsystems/indexer/IndexerIO.java (hunk 4 above)
Comment (mimizh2418) [reply src/main/java/org/team1540/robot2024/subsystems/indexer/IndexerIO.java:9]: yeah i'm just doing this to make this have more consistent formatting with the rest of the IOs, and it's nice to see the default values.

File: src/main/java/org/team1540/robot2024/subsystems/shooter/Shooter.java (hunk 5 above)
Comment (mimizh2418) [reply src/main/java/org/team1540/robot2024/subsystems/shooter/Shooter.java:263]: Each of these tables represents a different use case of publishing to NT and/or logs. AdvantageKit/RealOutputs/Shooter/Flywheels and AdvantageKit/Shooter/Flywheels are only for logging of inputs and outputs, and no user dashboard inputs should come from these. SmartDashboard/Shooter/Flywheels should only be used for values that would like to be changed conveniently at any point, whereas SmartDashboard/TunableNumbers/Shooter/Flywheels are for values that should be constant during normal code execution, but we want to have access to if we need to tune them.

**Pull request merged**